lib/Twig/Extensions/Node/Trans.php

```diff
@@ -37,14 +43,22 @@ public function __construct(Twig_Node $body, Twig_Node $plural = null, Twig_Node
      */
     public function compile(Twig_Compiler $compiler)
     {
+        // Reset default configuration from current environment
+        if (is_null($this->extension) && $compiler->getEnvironment()->hasExtension('Twig_Extensions_Extension_I18n')) {
+            $this->extension = $compiler->getEnvironment()->getExtension('Twig_Extensions_Extension_I18n');
+
+            $this->delims = $this->extension->getDelimiters();
```

Comment:
please don't do that. The TokenParser should instantiate the node with the proper attributes instead.